docs/concepts/advanced/html_fragments.md

```diff
@@ -1,5 +1,5 @@
-Django-components provides a seamless integration with HTML fragments ([HTML over the wire](https://hotwired.dev/)),
-whether you're using HTMX, AlpineJS, or vanilla JavaScript.
+Django-components provides a seamless integration with HTML fragments with AJAX ([HTML over the wire](https://hotwired.dev/)),
```

Comment:
I plan to review / update this page some time in the future, so I just updated the links / names